### PR TITLE
Migrate to Chef 14

### PR DESCRIPTION
**Pull request by dmgasper**

### CODE REVIEW

File: recipes/apc.rb [hunk 1]
```diff
@@ -25,18 +25,19 @@
...
 
-include_recipe 'build-essential'
-
 %w(httpd-devel pcre pcre-devel).each do |pkg|
   package pkg do
     action :install
   end
 end
 
-php_pear 'APC' do
```
Comment (ramereth) [inline recipes/apc.rb]: Is there a reason for changing the case of this?

File: recipes/apc.rb [hunk 2]
```diff
@@ -25,18 +25,19 @@
...
+build_essential 'apc'
+
+php_pear 'apc' do
   action :install
+  channel 'pecl'
```
Comment (ramereth) [inline recipes/apc.rb]: Is there a reason why we need to set the channel here? If so, can you please add a comment?

**Comment by ramereth** [comment]
[Chef 14: osl-php](https://trello.com/c/FgJdrrh1/1330-chef-14-osl-php)

**Comment by dmgasper** [comment]
!bump major ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v4.0.0.  Have a nice day!

**Pull request merged**